TrafficCapture/trafficCaptureProxyServer/src/main/resources/log4j2.properties

```diff
@@ -6,7 +6,9 @@ appender.console.type = Console
 appender.console.name = STDERR
 appender.console.target = SYSTEM_ERR
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = [%-5level] %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC} [%t] %c{1} - %msg%equals{ ctx=%mdc}{ ctx=\{\}}{}%n
+appender.console.layout.pattern = [%-5level] %d{DEFAULT_MICROS}{UTC} [%t] %c{1} - %msg%equals{ ctx=%mdc}{ ctx=\{\}}{}%n
+appender.console.direct = true
+appender.console.includeLocation = true
```

Comment:
Can you elaborate on including this/why it's important?

https://logging.apache.org/log4j/2.x/manual/async.html#location-location-location
> The [performance impact](https://logging.apache.org/log4j/2.x/performance.html#asyncLoggingWithLocation) of taking a stack trace snapshot is even higher for asynchronous loggers: logging with location is 30-100 times slower than without location.

Reply:
Removed, had originally included it due to the `%c{1}` in our layout pattern, though reading now that `%c` means logger name whereas `%C` means class which would have necessitated location